hathor/cli/run_node.py

```diff
@@ -117,6 +117,8 @@ def create_parser(cls) -> ArgumentParser:
                             help=f'Signal support for a feature. One of {possible_features}')
         parser.add_argument('--signal-not-support', default=[], action='append', choices=possible_features,
                             help=f'Signal not support for a feature. One of {possible_features}')
+        parser.add_argument('--x-asyncio-reactor', action='store_true',
```

Comment:
Maybe add it to the list of unsafe parameters?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/889/commits/af2bb36f82eef6f6c5fdd9d6e33ae99e355ae1be

Reply:
Thanks!